src/app/pages/RuntimeTransactionDetailPage/index.tsx

```diff
@@ -173,15 +174,17 @@ export const RuntimeTransactionDetailView: FC<{
                 }}
               >
                 {(transaction?.signers ?? []).map((signer, index) => (
-                  <>
+                  <Box key={`signer-${index}-link`} sx={{ display: 'inline-flex', alignItems: 'center' }}>
                     <AccountLink
-                      key={`signer-${index}`}
                       scope={transaction}
                       address={(signer.address_eth ?? signer.address) as string}
                     />
-                    <CopyToClipboard value={signer.address_eth ?? signer.address} />
+                    <CopyToClipboard
+                      key={`signer-${index}-copy`}
```

Comment:
(this shouldn't need a key now that parent has it)

Reply:
True. Removed.